src/Microsoft.TemplateEngine.Cli/TableOutput/HelpFormatter.cs

```diff
@@ -348,14 +355,24 @@ public TextWrapper(IEngineEnvironmentSettings environmentSettings, string text,
 
             public int MaxWidth { get; }
 
-            public string GetTextWithPadding(int line, int maxColumnWidth)
+            public string GetTextWithPadding(int line, int maxColumnWidth, bool rightAlign = false)
             {
                 var text = _lines.Count > line ? _lines[line] : string.Empty;
                 var abbreviatedText = ShrinkTextToLength(text, maxColumnWidth);
 
-                return
-                    abbreviatedText
-                    .PadRight(maxColumnWidth);
+
+                if (rightAlign)
+                {
+                    return
```

Comment:
Is this a common indentation style in this project? Otherwise, I suggest we simplify this by making this a single line.